content/en/docs/instrumentation/js/getting-started/browser.md

```diff
@@ -38,28 +38,50 @@ Copy the following file into an empty directory and call it `index.html`.
 
 ### Installation
 
+Let's initialize the the package and config files for the example project.
+
+{{< tabs TypeScript JavaScript >}}
+
+{{< tab >}}
+$ tsc --init
+{{< /tab >}}
+
+{{< tab >}}
+$ npm init -y
+{{< /tab >}}
+
+{{< /tabs >}}
+
```

Comment:
As I mentioned before, it doesn't make sense to use tabs here. For context, see the following thread: https://github.com/open-telemetry/opentelemetry.io/pull/1800#discussion_r989541203